src/main/java/de/wejago/mqtt2influx/service/Subscriber.java

```diff
@@ -0,0 +1,6 @@
+package de.wejago.mqtt2influx.service;
+
+import org.eclipse.paho.client.mqttv3.IMqttMessageListener;
+
+public interface Subscriber extends IMqttMessageListener {
```

Comment:
An extra interface is not needed here, let our subscribers just use `IMqttMessageListener`